addon/i3dio/ui/object.py

```diff
@@ -872,6 +893,42 @@ def draw(self, context):
         row.prop(obj.i3d_mapping, 'mapping_name')
 
 
+@persistent
+def check_lod_distance(dummy):
+    for obj in bpy.data.objects:
+        if obj.type == 'EMPTY':
+            if hasattr(obj, 'i3d_attributes') and 'lod_distance' in obj['i3d_attributes']:
+                current_lod = obj['i3d_attributes']['lod_distance']
+                # Skip if already converted
+                if not isinstance(current_lod, str):
+                    continue
+
+                # Check for placeholder or empty strings
+                if current_lod in ('', 'Enter your LOD Distances if needed.'):
+                    print(f"LOD distance for {obj.name} is not set.")
+                    continue

```

Comment:
This check probably isn't necessary as the try/catch will catch the case of it being a string and the empty case will just lead to `new_lod_distance` being default